datalab/datalab_session/data_operations/rgb_stack.py

```diff
@@ -107,10 +107,9 @@ def _align_images(self, fits_files: list[str]) -> list[str]:
                 aligned_images.append(aligned_img)
         
         if len(aligned_images) != self.REQUIRED_INPUTS:
-            raise ClientAlertException('Failed to align all images')
+            log.info('could not align all images')
```

Comment:
Maybe make this a warning and give some more identifying info in the log line? Like "Could not align images {image names here} in operation {operation id - name here}"?